### PR TITLE
TRIVIAL: fix typos

### PR DESCRIPTION
**Pull request by kabanoid**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/870/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/732/ : SUCCESS in 2m 09s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/999/ : SUCCESS in 2m 11s

File: lib/gooddata/models/blueprint/dataset_blueprint.rb [hunk 1]
```diff
@@ -144,7 +144,7 @@ def self.labels(dataset)
         find_columns_by_type(dataset, :label)
       end
 
-      def self.reference_label_for_attribtue(dataset, attribute)
+      def self.reference_label_for_attribute(dataset, attribute)
```
Comment (kubamahnert) [inline lib/gooddata/models/blueprint/dataset_blueprint.rb]: As all the SDK methods are in essence part of an API, we should provide backwards compatibility - there is either an `alias_method` functionality in ruby (IMO an uglier solution as the method definition would still have a typo) or we can simply define another method with the original name that just calls the new method and passes it all the params.

Comment (kabanoid) [reply lib/gooddata/models/blueprint/dataset_blueprint.rb]: Not sure if I got that properly: please see the fixup commit b272fbb76b70592c69e620c0f6941988979244da

File: lib/gooddata/models/blueprint/dataset_blueprint.rb [hunk 2]
```diff
@@ -144,11 +144,15 @@ def self.labels(dataset)
...
         labels = labels_for_attribute(dataset, attribute)
         labels.find { |label| label[:reference_label] == true } || labels.first
       end
 
+      def self.reference_label_for_attribtue(dataset, attribute)
+        self.reference_label_for_attribute(dataset, attribute)
```
Comment (yenkins) [inline lib/gooddata/models/blueprint/dataset_blueprint.rb]: Style/RedundantSelf: Redundant `self` detected.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/875/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/736/ : FAILURE in 39s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1003/ : SUCCESS in 40s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/876/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/737/ : SUCCESS in 45s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1004/ : SUCCESS in 34s

File: lib/gooddata/models/blueprint/dataset_blueprint.rb (hunk 1 above)
Comment (kubamahnert) [reply lib/gooddata/models/blueprint/dataset_blueprint.rb]: @kabanoid looks good; an even better approach would be with the splat operator to pass all params without explicitly mentioning them - https://alexcastano.com/everything-about-ruby-splats/#the-simple-splat . I'd also feel better with a comment that explains why we decided to have this method with a typo.

**Pull request closed without merging**